src/Orchard.Web/Modules/Orchard.Setup/Services/SetupService.cs

```diff
@@ -154,6 +155,10 @@ public class SetupService : ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
 
+                    // Create the distributed lock record schema.
+                    var distributedLockSchemaBuilder = new DistributedLockSchemaBuilder(_shellSettings, schemaBuilder);

```

Comment:
I am talking about the automatic migration class itself. Which do create the schema then try to acquire the lock.